runtime/RuntimeCommon.cpp

```diff
@@ -231,3 +231,7 @@ void symcc_make_symbolic(void *start, size_t byte_length) {
   _sym_make_symbolic(start, byte_length, inputOffset);
   inputOffset += byte_length;
 }
+
+SymExpr _sym_build_bit_to_bool(SymExpr expr) {
+  return _sym_build_equal(expr, _sym_build_integer(0, _sym_bits_helper(expr)));
```

Comment:
Maybe I just confused myself, but shouldn't this be a 1 (or "not equal")? If I call it with an expression representing, say, 0: 0 is equal to 0, so the equality would be true - that's not right, or is it? :thinking: